src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -131,23 +132,39 @@ def _create_tasks(
             (ContainerType.coverage, containers[ContainerType.coverage]),
             (ContainerType.readonly_inputs, containers[ContainerType.inputs]),
         ]
-        self.logger.info("creating libfuzzer_coverage task")
+        self.logger.info("creating coverage task")
+
+        # The `coverage` task is not libFuzzer-aware, so invocations of the target fuzzer
```

Comment:
We should detect an absent `{input}` in the service, or on the model.

Reply:
The subtlety is that it should _not_ be needed for targets that only read from `stdin`, but we don't actually support those yet.

Reply:
Wherever the checking is done, we should check that `{input}` is provided in `target_env` or `target_options`

Reply:
Yep, exactly.

Reply:
Captured in #925.